.github/workflows/ci-actions-incremental.yml

```diff
@@ -129,17 +131,31 @@ jobs:
       - name: Generate cache key
         id: cache-key
         run: |
+          CURRENT_BRANCH="${{ github.repository != 'quarkusio/quarkus' && 'fork' || github.base_ref || github.ref_name }}"
```

Comment:
What is `fork`?

Reply:
So this is the equivalent of the ternary operator in GitHub world.

In the case of a fork, we don't want to use the branch as the name of the cache, as you use feature branch. So for forks, we are just using `fork` as the equivalent of the branch name.